controls/radpdfviewer/features/find.md

```diff
@@ -0,0 +1,147 @@
+
+The **Find** method has two overloads:
+* **Find(string text)**: Finds next mach for the specified text in the current document.
+* **Find(string text, TextSearchOptions options)**: Finds next match for the specified text in the current document using specified options.
+
+#### __[C#] Example 1: Find specified text using specified options__
+
+{{region radpdfviewer-find_0}}
+
+    string sampleText = "sample text";
+    TextSearchOptions textSearchOptions = 
+        new TextSearchOptions(caseSensitive: false, useRegularExpression: false, wholeWordsOnly: true);
+
+    SearchResult searchResult = this.pdfViewer.Find(sampleText, textSearchOptions);
+{{endregion}}
+
+#### __[VB.NET] Example 1: Find specified text using specified options__
+
+{{region radpdfviewer-find_0}}
+
+    Dim sampleText As String = "sample text"
+    Dim textSearchOptions As TextSearchOptions = 
+        New TextSearchOptions(caseSensitive:=False, useRegularExpression:=False, wholeWordsOnly:=True)
+        
+    Dim searchResult As SearchResult = Me.pdfViewer.Find(sampleText, textSearchOptions)
+{{endregion}}
+
+The **FindAll** method has two overloads:
+* **FindAll(string text)**: Finds all matches for the specified text in the current document.
+* **FindAll(string text, TextSearchOptions options)**: Finds all matches for the specified text in the current document using specified options.
+
+#### __[C#] Example 2: Find all matches for the specified text using specified options__
+
+{{region radpdfviewer-find_1}}
+
+    IEnumerable<SearchResult> searchResult = this.pdfViewer.FindAll(sampleText, textSearchOptions);
+{{endregion}}
+
+#### __[VB.NET] Example 2: Find all matches for the specified text using specified options__
+
+{{region radpdfviewer-find_1}}
+
+    Dim searchResult As IEnumerable(Of SearchResult) = Me.pdfViewer.FindAll(sampleText, textSearchOptions)
+{{endregion}}
+
+The **FindPrevious** method has two overloads:
+* **FindPrevious(string text)**: Finds the previous text in the current document.
+* **FindPrevious(string text, TextSearchOptions options)**: Finds the previous text in the current document using specified options.
+
+#### __[C#] Example 3: Finds the previous text in the current document using specified options__
+
+{{region radpdfviewer-find_2}}
+
+    SearchResult searchResult = this.pdfViewer.FindPrevious(sampleText, textSearchOptions);
+{{endregion}}
+
+#### __[VB.NET] Example 3: Finds the previous text in the current document using specified options__
+
+{{region radpdfviewer-find_2}}
+
+    Dim searchResult As SearchResult = Me.pdfViewer.FindPrevious(sampleText, textSearchOptions)
+{{endregion}}
+
+## SearchResult
```

Comment:
Use h3 (###) to mark this section as it is a subsection of ## Find Programmatically